.changeset/two-gifts-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+"@logto/connector-logto-social-demo": major
```

Comment:
should be minor? since no breaking change

Reply:
updated